turbopack/crates/turbopack-ecmascript/src/references/amd.rs

```diff
@@ -123,12 +125,11 @@ pub enum AmdDefineFactoryType {
     Value,
 }
 
-#[turbo_tasks::value(shared)]
-#[derive(Debug)]
+#[derive(PartialEq, Eq, Serialize, Deserialize, TraceRawVcs, ValueDebugFormat, NonLocalValue)]
 pub struct AmdDefineWithDependenciesCodeGen {
     dependencies_requests: Vec<AmdDefineDependencyElement>,
     origin: ResolvedVc<Box<dyn ResolveOrigin>>,
-    path: ResolvedVc<AstPath>,
+    path: AstPath,
     factory_type: AmdDefineFactoryType,
     issue_source: ResolvedVc<IssueSource>,
```

Comment:
As follow-up PR: I think `IssueSource` doesn't need to be a Vc, neither need `SourceRange`.